sdk/parents/azure-client-sdk-parent/pom.xml

```diff
@@ -458,57 +458,57 @@
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-enforcer-plugin</artifactId>
         <version>3.0.0-M3</version> <!-- {x-version-update;org.apache.maven.plugins:maven-enforcer-plugin;external_dependency} -->
+        <configuration>
+          <rules>
+            <!--
+            Rule: https://maven.apache.org/enforcer/enforcer-rules/bannedDependencies.html
+            We have a list of blessed dependencies here: https://azure.github.io/azure-sdk/java_implementation.html#dependencies
+            Format of includes / excludes is the following: 'groupId[:artifactId][:version][:type][:scope][:classifier]'.
+            Wildcards can be used.
+            -->
+            <bannedDependencies>
+              <searchTransitive>false</searchTransitive>
+              <excludes>
+                <!-- We exclude everything that is runtime-time, compile-time, or provided scoped (i.e. we allow
+                dependencies when they are used for test scope or otherwise) -->
+                <exclude>*:*:*:*:runtime</exclude>
+                <exclude>*:*:*:*:compile</exclude>
+                <exclude>*:*:*:*:provided</exclude>
+              </excludes>
+              <includes>
+                <include>com.azure:*</include>
+
+                <!-- Used by many libraries to bring in annotations used by Reactor -->
+                <include>com.google.code.findbugs:jsr305</include>
```

Comment:
With my submitted PR this should already be tagged, though you're going to have to merge those changes with your.

Reply:
Done